src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -186,6 +186,28 @@ void onCommand(CommandExecuted event) {
         } else if ("DinkRegion".equalsIgnoreCase(cmd)) {
             int regionId = WorldUtils.getLocation(client).getRegionID();
             plugin.addChatSuccess(String.format("Your current region ID is: %d", regionId));
+        } else if ("DinkMigrate".equalsIgnoreCase(cmd)) {
+            if (args == null || args.length == 0) {
+                plugin.addChatWarning("Please specify which plugin's settings to migrate or 'all'. " +
+                    "Supported plugins include: " + String.join(", ", MigrationUtil.PLUGIN_METADATA.keySet()));
+            } else if (args.length > 1) {
+                plugin.addChatWarning("Please only specify one plugin at a time to migrate");
+            } else {
+                String key = args[0];
+                if ("all".equalsIgnoreCase(key)) {
+                    MigrationUtil.PLUGIN_METADATA.values()
+                        .forEach(func -> migrateConfig(func.apply(config)));
+                } else {
+                    var metadata = MigrationUtil.findMetadata(key, config);
+                    if (metadata == null) {
+                        plugin.addChatWarning("Failed to recognize plugin name to be migrated");
+                        return;
+                    }
+                    migrateConfig(metadata);
+                }
+                plugin.addChatSuccess("Finished migrating configs from other plugins. " +
+                    "Please verify the latest Dink settings and disable your other webhook plugins");
```

Comment:
Should we mention that you need to re-open config panel for changes to be reflected?

Reply:
`Please close and open the plugin settings panel for these changes to be visually reflected.` should already be in chat from the `handleImport` call, but we could reiterate it since ppl struggle to read

Reply:
ah if that's already there from the import call, then we should be all good